shared/js/background/https.es6.js

```diff
@@ -147,10 +147,8 @@ class HTTPS {
             return reqUrl
         }
 
-        // Skip upgrading sites that have been whitelisted by user
-        // via on/off toggle in popup
-        if (tab.site.whitelisted) {
-            console.log(`HTTPS: ${tab.site.domain} was whitelisted by user. Skip upgrade check.`)
```

Comment:
Removed the logging, I'm not sure it's important here and we're not checking the feature or if the site's on the global allowlist also.